### PR TITLE
Jgfouca/upstream merge 2022 06 02

### PR DESCRIPTION
**Pull request by jgfouca**
No conflicts.

I noticed this problem:
`FAIL ERS_Ln9.ne30_ne30.F2010-SCREAMv1.mappy_gnu9 (phase COMPARE_base_rest)`

But this appears to be on master too.

### CODE REVIEW

**Comment by sarats** [comment]
We have some customized settings for getting things running on Summit which will be overridden by upstream, so need to manage changes to config_machines.xml.

File: cime_config/machines/config_machines.xml [hunk 1]
```diff
@@ -3492,6 +3492,8 @@
     <MAX_TASKS_PER_NODE compiler="pgigpu">18</MAX_TASKS_PER_NODE>
     <MAX_TASKS_PER_NODE compiler="gnugpu">6</MAX_TASKS_PER_NODE>
     <MAX_TASKS_PER_NODE compiler="ibmgpu">42</MAX_TASKS_PER_NODE>
+	<!-- Need to activate following attribute after CIME update from upstream -->
```
Comment (sarats) [inline cime_config/machines/config_machines.xml:3495]: Did I miss something? Are these 2 lines the only changes to the machine config?

Comment (jgfouca) [reply cime_config/machines/config_machines.xml:3495]: Yes, it seems upstream did not do much to modify config_machines.xml since the last update.

Comment (jgfouca) [reply cime_config/machines/config_machines.xml:3495]: Wasn't the config_machines change made to both SCREAM and E3SM? In that case, the change would already be in SCREAM and so would not show up here.

Comment (PeterCaldwell) [reply cime_config/machines/config_machines.xml:3495]: I think Sarat's confusion is that he merged SCREAM PR #1690 the other day (https://github.com/E3SM-Project/scream/pull/1690/files) and it included a lot of changes to config_machines.xml which I would have thought would show up as diffs here. But maybe I'm just not understanding git merge.

Comment (bartgol) [reply cime_config/machines/config_machines.xml:3495]: Yeah, I don't think that matters. The diffs you see here are the changes that _these_ commits add to current master. Whether upstream/HEAD and scream/HEAD had _lots_ of diffs is a separate issue.

Comment (sarats) [reply cime_config/machines/config_machines.xml:3495]: Yes, my changes were the only ones to this file in both SCREAM and E3SM and looks like the SCREAM-specific changes survive (MAX_MPI_TASKS "gnugpu" = 6). So we're good.

**Review by sarats — APPROVED** [review]

**Pull request merged**